tests/general_generators.py

```diff
@@ -33,6 +33,10 @@ def any_past_datetime_string() -> str:
     return any_past_datetime().isoformat()
 
 
+def any_past_datetime_string_without_offset() -> str:
```

Comment:
https://github.com/linz/geostore/pull/1582#discussion_r863115795 is relevant here as well.